### DISCUSSION TITLE
Suppression du système de fight classé

### DISCUSSION THREAD

**Discussion by Thesmokingllama** [Crownicles/Crownicles#2337] [Suggestions fermées]
Argument 1 : comme vous le savez, il y a du feedfight et il y en a toujours eu, or, si on ne peut pas changer les joueurs et le système a toujours été défaillant, plutôt qu'essayer de résoudre l'insolvable supprimons simplement le problème

Argument 2 : le système de fight classé favorise les joueurs haut level au détriment des plus faibles. Autant c'est cool quand on est haut level de pouvoir drop du stuff rare, autant quand on débute c'est vite déprimant de constater qu'on pourra jamais rattraper les dominants vu que l'écart ne fait que se creuser parce qu'ils ont l'habitude de jeu et de meilleures récompenses (ce qui incite les moins développés à feedfight pour espérer se développer)

Argument 3 : Les prouesses guerrières des joueurs sont maintenant aussi récompensées avec l'île, île sur laquelle je le rappelle on ne peut pas vraiment cheat

Argument 4 : le systeme de fight classé a rendu les armes de rareté supérieure banales

**Comment by Easygamer01s** [comment]
Pour l'argument 2, c'est tout le contraire justement.. plus les gains/temps sont augmentés, plus l'historique de jeu baisse en importance, plus les nouveaux joueurs sont proches des anciens.. si tu donnes 2k XP par semaine en plus, c'est des jours de progression qui ne sont plus à faire..

Pour l'argument sur la rareté des items, je suis d'accord avec toi, maintenant c'est conditionné par un minimum d'activité, quelques matchs par semaine pour se maintenir..

À mon sens, si tu supprimes le classé, le pvp est mort, ce qui serait vraiment dommage car, malgré certains comportements, c'est vraiment une grosse partie du fun de ce jeu, et un point de rencontre fantastique en les joueurs.. :) 

Il y a eu des soucis de feedfight il y a quelque temps avec la guilde pourpre et associés.. aujourd'hui je ne suis pas certain d'en voir les conséquences..

**Reply by Thesmokingllama** [reply]
J'ai pas bien compris en quoi l'argument 2 c'est tout le contraire. Si l'historique de jeu baisse en importance pour les débutants c'est aussi notre cas. D'autant plus que les 2k d'xp, c'est nous qui les obtenons, pas les débutants qui obtiennent des récompenses dérisoires en comparaison.

En ce qui concerne la suppression du fight classé vraiment pas convaincue que ça tue le pvp, il reste les missions de fight pour maintenir en vie la pratique et au pire on pourrait imaginer intégrer plus tard le système de matchmaking dont Draft nous parle depuis maintenant un moment.

**Reply by Easygamer01s** [reply]
J'ai l'impression que les conditions ne sont pas remplies pour un matchmaking bien appliqué.. trop peu de joueurs pour ça, pas assez fluide.. bien que je me suis déjà demandé à quoi pourrait ressembler l'arène si on retirait l'avantage du level.. on pourrait pratiquement jouer tous ensemble..

On pourrait imaginer des missions quotidiennes basées sur des matchs gagnés, moins de feedfight et un bonus à venir en arène..

A mon sens, c'est un gros challenge de draftbot de trouver un équilibre entre éviter les comportements nuisibles et créer un jeu fun.. je ne suis pas certain que supprimer le classé soit une bonne idée ..

**Reply by Easygamer01s** [reply]
Pour l'argument 2, imagine que l'on reçoit tous au quotidien 2 millions d'xp, toute ton évolution n'aurait plus d'importance..
Avec ce nouveau système de bonus de ligue, on a rendu l'évolution plus facile, rapprochant le nouveau joueur de l'ancien.. globalement, tout nouveau bonus avantage le nouveau joueur

**Comment by Voisin1234** [comment]
Je comprends les préoccupations exprimées, mais supprimer le système de fight classé pourrait priver les joueurs d'un aspect compétitif. Des ajustements pourraient être envisagés pour équilibrer les récompenses et encourager la progression des joueurs moins expérimentés, plutôt que de le supprimer entièrement.

**Reply by Voisin1234** [reply]
on pourrait par exemple faire déjà des vrais récompenses pour chaque ligue et non pour le système entier, après, on peut faire un système de classe général qui ne changera jamais comme le classement normale et un classement glorieux toutes les semaines remis à zéro comme le TW on pourrait faire un système de classement avec le nombre de combats gagné par semaine ou par le nombre de points rapportés( le problème avec les points, c'est que les petits en gagnent beaucoup) Je ne sais pas comment gérer votre double caché, mais pour moi étant petit niveau moi qui viens de recommencer le jeu, franchement je n’ai pas du tout ce point de vue. Je trouve que je monte assez vite que les grands sauront rattrapables dans six mois  je pense car vous êtes tellement haut que vous avancez moins vite que les petits arrive en force . Je parlerai feed mais bon on va rester sur une bonne journée. 👍
